src/frontend/components/DashboardDocument/index.tsx

```diff
@@ -82,6 +82,10 @@ const DashboardDocument = (props: DashboardDocumentProps) => {
     }
   }, []);
 
+  useEffect(() => {
```

Comment:
IMO it is not a good practice to change the value of an input once it has been rendered and the user has seen it.

Maybe we should use `document.title` for the displayed value in the top right hand corner and leave `title` for the form.

Reply:
Sorry but I don't understand what you mean. the `title` variable initialized by `useState` is only used in the form and we have to update it if the document title is updated from somewhere else. What is the _displayed value in the top right hand corner_ ?

Reply:
The title is shown in the top right too:
<img width="1021" alt="Capture d’écran 2019-09-17 à 16 02 57" src="https://user-images.githubusercontent.com/1932937/65048594-a9309080-d964-11e9-9f99-bcd5dc77e09b.png">

IMO the rule of thumb is we should never update the content of an input after we have shown it to the user. It's jarring for users and not the way a form should behave

Reply:
But who is updating it? Are we getting a new version from the server? If this is it I think the form should still show the old version...

Reply:
Yes the server update it when the `update-state` is called from AWS. A new title is set and the poll function fetch a new object with a new title. This is the first comment we had when d2rp tried the document feature.

Reply:
The bug happen when the document state changed from processing to ready.

Reply:
In that case I guess the way to handle this is to not initialize the form title and not show the form until the document is ready?

Reply:
It is already the case expect for the form title initialization. Creating a state in a condition is not really a good practice or I am wrong ?

Reply:
> Creating a state in a condition is not really a good practice
React with actually prevent you from doing this.

IMO this just means we need a small component for the form, that handles its own state.